lab_1_keywords_tfidf/main.py

```diff
@@ -4,20 +4,31 @@
 """
 from typing import Optional, Union
 
+from lab_1_keywords_tfidf.assets.freq_and_idf_dictionary_creation import file
+
+
+def clean_and_tokenize(text: str) -> [list[str]]:
+    name_file = 'Дюймовочка.txt'
+    File = open(name_file, 'rt')
+    text = File.read()
+    File.close()
+    words = text.split()
+    words = [word.lower() for word in words]
+    print(words)
+    return(words)
```

Comment:
Don't forget to check whether the text that was passed to the function has correct format
If it happens to be corrupt, None is expected to be returned